CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## TBD
 
+### Enhancements
```

Comment:
I don't think I'd class either of these as an enhancement for Android - patch release seems fine.